common/src/api/external/mod.rs

```diff
@@ -3408,6 +3408,10 @@ pub struct TufArtifactMeta {
 
     /// The size of the artifact in bytes.
     pub size: u64,
+
+    /// Sign (root key hash table) hash of a signed image. Only applicable to
+    /// RoT or RoT bootloader artifacts.
+    pub rot_sign: Option<Vec<u8>>,
```

Comment:
```suggestion
    /// Contents of the `SIGN` field of a Hubris archive caboose, i.e.,
    /// an identifier for the set of valid signing keys. Currently only
    /// applicable to RoT image and bootloader artifacts, where it will
    /// be an LPC55 Root Key Table Hash (RKTH).
    pub sign: Option<Vec<u8>>,
```